src/style.css

```diff
@@ -0,0 +1,36 @@
+
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: center;
```

Comment:
```suggestion
  background: url(images/star.svg) no-repeat center auto;
```

Reply:
Hi, `background: url(images/star.svg) no-repeat center auto;` for some reason this doesn't work.